packages/dialect-react-ui/components/Notifications/index.tsx

```diff
@@ -85,38 +90,52 @@ function CreateThread() {
   );
 }
 
-function Settings(props: { toggleSettings: () => void }) {
+function Settings(props: {
+  toggleSettings: () => void;
+  notifications: NotificationType[];
+}) {
   const { dialectAddress, deleteDialect, isDialectDeleting, deletionError } =
     useDialect();
   const { colors, textStyles, icons } = useTheme();
 
   return (
     <>
       <div className="mb-3">
-        <p className={cs(textStyles.body, 'mb-1')}>Included event types</p>
-        <ul className={cs(textStyles.bigText, 'list-disc pl-6')}>
-          <li>Welcome message on thread creation</li>
-        </ul>
+        <h2 className={cs(textStyles.h2, 'mb-1')}>Notifications</h2>
+        {props.notifications?.map((type) => (
+          <ValueRow
+            key={type.name}
+            label={type.name}
+            highlighted
+            className={cs('mb-1')}
+            colon={false}
```

Comment:
What is the colon for?

Reply:
Sorry didn't quite understand the use-case of the colon. I don't mind leaving it, just don't quite understand it. Could you please provide an example?

Reply:
Also, since it's an optional prop, no need to provide it here explicitly.

Reply:
It seems now it become useless, since in new designs we have only one look of ValueRow